### PR TITLE
@zephraph => Add <Footer />

### PR DESCRIPTION
**Pull request by damassi**
footer

### CODE REVIEW

File: src/Styleguide/Components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            <Link href="#">Consign with Artsy</Link>
+            <Link href="#">Artsy for Professional Buyers</Link>
+          </Serif>
+        </Flex>
+      </Flex>
+    </React.Fragment>
+  )
+}
+
+const Link = styled.a`
```
Comment (zephraph) [inline src/Styleguide/Components/Footer.tsx]: It begins 😛

File: src/Styleguide/Components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  }
+}
+
+export const LargeFooter = props => <FooterContainer flexDirection="row" />
+export const SmallFooter = props => <FooterContainer flexDirection="column" />
+
+export interface FooterContainerProps extends FlexDirectionProps {}
+const FooterContainer: React.SFC<FooterContainerProps> = props => {
+  return (
+    <React.Fragment>
```
[the remark this answers is not in the document]
Comment (damassi) [reply src/Styleguide/Components/Footer.tsx]: Typo actually, will remove

**Comment by damassi** [comment]
(@oxaudo Reassigning as it looks you've left for day)

**Comment by zephraph** [comment]
LGTM 👍 

Feel free to merge on green.

**Pull request merged**